### PR TITLE
Cleanup: deflake jax.experimental and jax.ops

### PR DESCRIPTION
**Pull request by jakevdp**

### CODE REVIEW

**Comment by jakevdp** [comment]
Oops, looks like the tabs-to-spaces conversion left some documentation alignments incorrect. I'll fix soon.

File: jax/ops/__init__.py [hunk 1]
```diff
@@ -12,6 +12,9 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+__all__ = [
```
Comment (hawkinsp) [inline jax/ops/__init__.py:15]: What is the value in setting `__all__` if we have already carefully crafted the namespace to contain nothing but exported symbols? It seems like it is just adding another place to update for no value.

Comment (jakevdp) [reply jax/ops/__init__.py:15]: This is the solution [recommended by PEP8](https://www.python.org/dev/peps/pep-0008/#id50), though I only tend to use it in cases where there aren't too many imports. If you feel strongly about this, I'm happy to change to a ``noqa: F401`` comment instead.

Comment (hawkinsp) [reply jax/ops/__init__.py:15]: I would prefer to leave it out; it just adds busy-work for whoever adds a function. We already created a facade module to solve the visibility problem.

Comment (jakevdp) [reply jax/ops/__init__.py]: OK, Done.

**Review by hawkinsp — APPROVED** [review]

**Pull request merged**